cmd/govim/internal/vimconfig/vimconfig.go

```diff
@@ -66,3 +66,16 @@ func BoolVal(v bool) *bool {
 func MapVal(v map[string]interface{}) *map[string]interface{} {
 	return &v
 }
+
+// EqualBool returns true iff i and j are both nil, or if both are non-nil and
```

Comment:
typo - `iff`

Reply:
incorrect comment, "returns true if i and j are both nil" and then `return false` if both are nil.

Reply:
I intentionally used `iff` there: https://en.wikipedia.org/wiki/If_and_only_if

Thanks - the code was actually wrong here 😢 - fixed.